src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -564,6 +690,109 @@ def _batchDeletionSize(self):
         from toil.jobStores.aws.jobStore import AWSJobStore
         return AWSJobStore.itemsPerBatchDelete
 
+    def _importFileFromKeyOfSize(self, size):
+        keyName = self.__uploadToBucketKeyOfSize(size).name
+        srcUrl = 's3://%s/%s' % (self.srcBucket.name, keyName)
+        JobStoreFileID = self.master.importFile(srcUrl)
+        self.assertEqual(self._hashKey(keyName), self._hashJobStoreFileID(JobStoreFileID))
+        return JobStoreFileID
+
+    def _hashFile(self, localFilePath):
+        totalSize = os.path.getsize(localFilePath)
+        with open(localFilePath, 'r') as f:
+            if totalSize <= 2**20 * 5:
```

Comment:
Isn't this equivalent to the partSize variable? If you define it at the beginning of the function then you can avoid this expression. Also, it may make sense to pull the default partSize out into a class variable so you can avoid redefining it over and over